internal/kafka/internal/handlers/validation_test.go

```diff
@@ -79,10 +79,10 @@ func Test_Validation_validateKafkaClusterNameIsUnique(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			gomega.RegisterTestingT(t)
+			RegisterTestingT(t)
 			validateFn := ValidateKafkaClusterNameIsUnique(&tt.arg.name, tt.arg.kafkaService, tt.arg.context)
 			err := validateFn()
-			gomega.Expect(tt.want).To(gomega.Equal(err))
+			Expect(tt.want).To(Equal(err))
```

Comment:
Parameter of `Expect` is the **actual** value, not the *expected one*.
This should be:
```suggestion
			Expect(err).To(Equal(tt.want))
```

Reply:
I'll reverse where needed. Thanks @ziccardi